raiden/transfer/channel.py

```diff
@@ -57,18 +69,28 @@
     ContractReceiveChannelSettled,
     ContractReceiveChannelWithdraw,
     ReceiveTransferDirect,
+    ReceiveUnlock,
 )
 from raiden.utils import publickey_to_address, typing
 from raiden.settings import DEFAULT_NUMBER_OF_CONFIRMATIONS_BLOCK
 
+# This should be changed to `Union[str, MerkleTreeState]`
```

Comment:
When should that be done? Maybe add a `TODO` as well.

Reply:
Any time really, I didn't change it because the current code works and it's not really a priority to improve this return type.